plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/QueryBuilder.java

```diff
@@ -90,6 +90,10 @@ public PreparedQuery prepareQuery(
         return new PreparedQuery(sql, accumulator.build());
     }
 
+    /**
+     * @deprecated Use #prepareSql and #prepareStatement instead.
```

Comment:
I don't get it.

Reply:
What is not clear here? I am saying that `buildSql` should not be used anymore.

Reply:
reworded

Reply:
taking it back; this is merged already